app/javascript/components/FetchingBoundary.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```

Comment:
This is a generic wrapper component that can be used when we're fetching data from an endpoint.